### PR TITLE
fix: add time picker with locale example

### PR DESCRIPTION
**Pull request by stefanoScalzo**
#### Please provide a link to the associated issue.
fixes: https://github.com/SAP/fundamental-ngx/issues/2825
#### Please provide a brief summary of this pull request.
This pr will add an example for the locale on time picker
It parses moment js in the typescript file to determine how to format the text
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by mikerodonnell89** [comment]
Possibly out of the scope of this PR but do the "Hours / Minutes / Seconds" labels need to change when changing locale?

**Comment by stefanoScalzo** [comment]
@mikerodonnell89 Definatly needs to be done but will have to be done in a separate PR

**Comment by salarenko** [comment]
Could you improve example design?
image

**Comment by InnaAtanasova** [comment]
> @mikerodonnell89 Definatly needs to be done but will have to be done in a separate PR

In this case create a ticket with the missing functionalities and link it here.

**Comment by JKMarkowski** [comment]
@stefanoScalzo I added 1 commit to this PR solving @InnaAtanasova comment. Could you check the behaviour now?

File: libs/core/src/lib/time/time.component.ts [hunk 1]
```diff
@@ -5,15 +5,15 @@ import {
     forwardRef,
     Input,
     OnChanges,
-    OnInit,
-    SimpleChanges,
+    OnInit, QueryList,
+    SimpleChanges, ViewChildren,
```
Comment (JKMarkowski) [inline libs/core/src/lib/time/time.component.ts:9]: Formatting

File: libs/core/src/lib/time/time.component.ts [hunk 2]
```diff
@@ -102,6 +102,9 @@ export class TimeComponent implements OnInit, OnChanges, ControlValueAccessor {
     @Input()
     spinnerButtons = true;
 
+    @ViewChildren(TimeColumnComponent)
```
Comment (JKMarkowski) [inline libs/core/src/lib/time/time.component.ts:105]: Let's add `@hidden` comment to hide it form API

**Pull request merged**